colormap.py

```diff
@@ -327,7 +327,8 @@
     '_': 'White',
     '#': 'Gray_D',
     'account_not_colored': 'reset',
-    'NoPattern': 'White'
+    'NoPattern': 'White',
+    'Pending': 'Yellow'
```

Comment:
This was the easy way of making Pending yellow in Queues. The only problem is, you get "Pending: R + Q" in yellow and not just the word Pending. 
Which could be considered a plus, depending on the way you see it.